backend/base/templates/signup.html

```diff
@@ -1,118 +1,97 @@
+                    {% endif %}
+                  </div>
+                  <div class="form-group">
+                    <label for="{{ form.phone_number.id_for_label }}">Phone number</label>
+                    {{ form.phone_number }}
+                    {% if form.phone_number.errors %}
+                      <div class="error">
+                        {% for error in form.phone_number.errors %}
+                          <p>{{ error }}</p>
+                        {% endfor %}
+                      </div>
+                    {% endif %}
+                  </div>
+                  <div class="text-center">
+                    <button type="submit" class="btn btn-primary btn-block enter-btn">Sign Up</button>
+                  </div>
 
-</head>
+                  <div class="d-flex">
+                    <button class="btn btn-facebook col mr-2">
+                      <i class="mdi mdi-facebook"></i> Facebook
+                    </button>
+                    <button class="btn btn-google col">
+                      <i class="mdi mdi-google-plus"></i> Google plus
+                    </button>
+                  </div>
 
-<body>
-  <div class="container-scroller">
-    <div class="container-fluid page-body-wrapper full-page-wrapper">
-      <div class="row w-100 m-0">
-        <div class="content-wrapper full-page-wrapper d-flex align-items-center auth login-bg">
-          <div class="card col-lg-4 mx-auto">
-            <div class="card-body px-5 py-5">
-              <h1 class="mt-4 mb-4" style="margin-left: 35%;">SIGN UP</h1>
-              <form method="post">
-                {% csrf_token %}
-                <div class="form-group mt-3 mb-3">
-                  <label for="{{ form.first_name.id_for_label }}">First Name</label>
-                  {{ form.first_name }}
-                </div>
-                <div class="form-group">
-                  <label for="{{ form.last_name.id_for_label }}">Last Name</label>
-                  {{ form.last_name }}
-                </div>
-                <div class="form-group">
-                  <label for="{{ form.organization_name.id_for_label }}">Organization Name</label>
-                  {{ form.organization_name }}
-                </div>
-                <div class="form-group">
-                  <label for="{{ form.email.id_for_label }}">Email</label>
-                  {{ form.email }}
-                  {% if form.email.errors %}
-                    <div class="error">
-                      {% for error in form.email.errors %}
-                        <p>{{ error }}</p>
-                      {% endfor %}
-                    </div>
-                  {% endif %}
+                  <p class="sign-up text-center">Already have an Account?<a href="{% url 'login' %}">Login</a></p>
+                  <!-- <p class="terms">By creating an account you are accepting our<a href="{% url 'termsandconditions' %}"> Terms & Conditions</a></p> -->
+                </form>

```

Comment:
comments can be removed

Reply:
That link can be viewed only through logging in making it redundant. Have to change it.